src/transformers/models/tvlt/feature_extraction_tvlt.py

```diff
@@ -0,0 +1,343 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Feature extractor class for TVLT."""
+
+from typing import List, Optional, Union
+
+import numpy as np
+from numpy.fft import fft
+
+from ...feature_extraction_sequence_utils import SequenceFeatureExtractor, BatchFeature
+from ...image_utils import is_torch_tensor
+from ...utils import TensorType, is_vision_available, is_speech_available, logging
+
+
+logger = logging.get_logger(__name__)
+
+if is_speech_available():
+    import torchaudio
+
+    
+class TvltFeatureExtractor(SequenceFeatureExtractor):
+    r"""
+    Constructs a TVLT audio feature extractor. This feature extractor can be used to prepare audios for the model.
+
+    This feature extractor inherits from [`FeatureExtractionMixin`] which contains most of the main methods. Users
+    should refer to this superclass for more information regarding those methods.
+
+    Args:
+
+    """
+
+    model_input_names = ["audio_values", "audio_masks"]
```

Comment:
cc @ArthurZucker I think the recommend name for audio models is `input_features` (and `attention_mask` ?)